ext/phar/phar_object.c

```diff
@@ -3055,6 +3061,12 @@ PHP_METHOD(Phar, getSignature)
 			case PHAR_SIG_OPENSSL:
 				add_assoc_stringl(return_value, "hash_type", "OpenSSL", 7);
 				break;
+			case PHAR_SIG_OPENSSL_SHA256:
+				add_assoc_stringl(return_value, "hash_type", "OpenSSL_SHA256", 14);
+				break;
+			case PHAR_SIG_OPENSSL_SHA512:
+				add_assoc_stringl(return_value, "hash_type", "OpenSSL_SHA512", 14);
```

Comment:
```suggestion
				add_assoc_string(return_value, "hash_type", "OpenSSL_SHA512");
```
...would avoid the hardcoded string length

Reply:
done (fox new entry, and old ones)